File: python/3strings/1st.py
'''
Write a Python program to remove special symbols/Punctuation from a given string.
'''

# define punctuation
pun = '''\?/<>.,;;"'{}[]()_+-*&^%$#@!~'''
print(pun)
# To take input from the user
str = input("enter the string :")  # hey !!! how,, are yo ?? or "\sachin is @criketer && a kind person"

# remove punctuation from the string
no_pun = " "
for char in str:
    if char not in pun:
        no_pun = no_pun + char

# display the unpunctuated string
print(no_pun)

File: c++/c++ clg assignments/1/1st.cpp
/*
Write a C++ program to display factors of a number.
*/

#include<iostream>

using namespace std;

int main()
{
    int n;
    cout << "\n enter a positive  numberr :";
    cin >> n;
    cout << "the factors of " << n << ":";
    for (int i = 1; i <= n; i++)
    {
        if(n % i == 0){
            cout << i << "\t";
        }
    }
    
    return 0;
}